### PR TITLE
Add tests for ClearCommand #327

### PR DESCRIPTION
**Pull request by Zhiyuan-Amos**
Part of #327

### CODE REVIEW

**Comment by CanIHasReview** [comment]
# v1

@Zhiyuan-Amos submitted v1 for review.

* [v1 1/1] [Add tests for ClearCommand](https://github.com/se-edu/addressbook-level4/pull/475/commits/33662fc7d8e9dfd82eb2d1911a3cb25eb4ceb64d)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/3de3b34bcf05aa4a9a656b238be22d7de6d78279...33662fc7d8e9dfd82eb2d1911a3cb25eb4ceb64d))
<details>
<summary>Checkout this PR version locally</summary>

```sh
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/475/1/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you wish to fetch this PR to.

</details>

File: src/test/java/seedu/address/logic/commands/ClearCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        assertCommandSuccess(model);
+    }
+
+    @Test
+    public void execute_nonEmptyAddressBook_success() {
+        Model model = new ModelManager(new TypicalPersons().getTypicalAddressBook(), new UserPrefs());
+        assertCommandSuccess(model);
+    }
+
+    private void assertCommandSuccess(Model model) {
```
Comment (yamgent) [inline src/test/java/seedu/address/logic/commands/ClearCommandTest.java]: Header comment for this method?

File: src/test/java/seedu/address/logic/commands/ClearCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        ClearCommand command = new ClearCommand();
+        command.setData(model, new CommandHistory());
+        command.execute();
```
Comment (yamgent) [inline src/test/java/seedu/address/logic/commands/ClearCommandTest.java]: we need to check for `feedbackToUser`.

**Comment by CanIHasReview** [comment]
# v2

@Zhiyuan-Amos submitted v2 for review.

* [v2 1/1] [Add tests for ClearCommand](https://github.com/se-edu/addressbook-level4/pull/475/commits/dbfbf6a6df3a3512357f6bb3b2c2695545c68548)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/3de3b34bcf05aa4a9a656b238be22d7de6d78279...dbfbf6a6df3a3512357f6bb3b2c2695545c68548)) ([:chart_with_upwards_trend: Interdiff between v1 and v2](https://github.com/se-edu/addressbook-level4/compare/2bf51c2dda51746e0c7eb835097a9a08e9608763~1...2bf51c2dda51746e0c7eb835097a9a08e9608763))
<details>
<summary>Checkout this PR version locally</summary>

```sh
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/475/2/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you wish to fetch this PR to.

</details>

**Pull request merged**